lms/djangoapps/ci_lrs/tests.py

```diff
@@ -0,0 +1,50 @@
+import random
+from unittest.mock import patch, MagicMock
+
+from django.conf import settings
+from django.utils import timezone
+from django.test import TestCase, override_settings
+
+from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure, DuplicateKeyError, NetworkTimeout, OperationFailure, ServerSelectionTimeoutError
+
+from ci_lrs.utils import write_lrs_record_to_mongo
+
+
+class TestLRSTasks(TestCase):
+    def setUp(self):
+        self.mongodb = MongoClient('mongodb://mongodb:27017')['test_db']
+        self.mongodb['student_activity'].delete_many({})
+
+    def test_write_lrs_record_to_mongo(self):
+        context = {
+            'activity_time': timezone.now().isoformat(),
```

Comment:
Also also - is there a activity_time field we can read rather than inferring now()? Just for those times when the queue gets backed up...

Reply:
Maybe if we put the now() at the point where it calls the task.

Reply:
Yes, the time is passed into the task at the point of creation to avoid wrong timestamps if the task is sitting in the queue for a bit.